pkg/util/testing.go

```diff
@@ -86,14 +119,34 @@ func NewFailedPlanUnknown() mock.Response {
 // by the client.
 func AppendTrackResponses(res ...mock.Response) []mock.Response {
 	var response = append(res,
-		PlanNotFound,
-		PlanNotFound,
-		NewSuccessfulPlan(),
+		NewSuccessfulCurrentPlan(),
+		NewSuccessfulCurrentPlan(),
+		NewSuccessfulCurrentPlan(),
 	)
 
 	return response
 }
 
+// NewMockTrackChangeParams creates new tracking params for test purposes.
+func NewMockTrackChangeParams(id string) planutil.TrackChangeParams {
```

Comment:
This exported function needs a test :)

Reply:
This function can't really be unit tested as it is, besides it's tested on many other test functions which test the plan tracking.